### PR TITLE
增加头尾相连循环滚动模式

### PR DESCRIPTION
**Pull request by chipanda**

### CODE REVIEW

File: DynamicText/index.js [hunk 1]
```diff
@@ -155,22 +217,25 @@ DynamicText.propTypes = {
   // 文字循环模式，默认reverse
   // reverse：轮转到末尾后再轮转回开头
   // restart: 轮转到末尾后返回至开头重新循环
```
Comment (dragonwong) [inline DynamicText/index.js]: 添加 `cycle` 的文档说明

Comment (chipanda) [inline DynamicText/index.js]: done

File: Example/src/page/DynamicText/index.js [hunk 2]
```diff
@@ -78,20 +78,23 @@ class Page extends Component {
...
               speed={10}
             >
               {this.state.text1}
             </DynamicText>
           </Article>
-          <Article titleStyle={styles.articleTitle} title="测试">
+          <Article titleStyle={styles.articleTitle} title="restart">
             <DynamicText
+              mode="restart"
+              delayTime={1000}
```
Comment (dragonwong) [inline Example/src/page/DynamicText/index.js]: There is a bug when line 96 & 97 are removed.

Comment (dragonwong) [inline Example/src/page/DynamicText/index.js]: There is a bug when line 96 & 97 are removed.

**Pull request closed without merging**